crates/ruff_linter/src/linter.rs

```diff
@@ -1064,6 +1064,97 @@ mod tests {
         PythonVersion::PY310,
         "MultipleCaseAssignment"
     )]
+    #[test_case(
+        "irrefutable_case_pattern_wildcard",
+        "
+        match value:
+            case _:
+                pass
+            case 1:
+                pass
+        ",
+        PythonVersion::PY310,
+        "IrrefutableCasePattern"
+    )]
+    #[test_case(
+        "irrefutable_case_pattern_capture",
+        "
+        match value:
+            case irrefutable:
+                pass
+            case 1:
+                pass
+        ",
+        PythonVersion::PY310,
+        "IrrefutableCasePattern"
+    )]
+    #[test_case(
+        "single_starred_assignment",
+        "
+        *a = [1, 2, 3, 4]
+        ",
```

Comment:
nit: I'd probably put these on one line if they fit, but it's not a big deal either way.

```suggestion
        "*a = [1, 2, 3, 4]",
```